README.md

```diff
@@ -90,6 +90,7 @@ Validator                               | Description
 --------------------------------------- | --------------------------------------
 **contains(str, seed [, options])**    | check if the string contains the seed.<br/><br/>`options` is an object that defaults to `{ ignoreCase: false, minOccurrences: 1 }`.<br />Options: <br/> `ignoreCase`: Ignore case when doing comparison, default false.<br/>`minOccurences`: Minimum number of occurrences for the seed in the string. Defaults to 1.
 **equals(str, comparison)**             | check if the string matches the comparison.
+**isAbaRouting(str)**               | check if the string is an ABA routing number for US bank account / cheque.
```

Comment:
BTW, this is just for US only? Do we plan to support other countries and should we have a locale param then?
Sorry missed this in my previous review.

Reply:
Yes, [ABA](https://www.aba.com/about-us/routing-number) is the abbv for American Bankers Association. The ABA number is used with every US bank account and cheque, to identify the corresponding financial institution. For example:

![Screenshot_Amex.jpg](https://user-images.githubusercontent.com/98693645/218268247-795cf753-e11e-4827-9136-fea1c114f1f7.jpg)

There is no need to add the locale param.

Reply:
Does any other country have a similar thing perhaps just with a different name? Trying to see if there's a way we could generalize here than having a whole validator just for one country...

Reply:
Great idea. I'll look into it. 😁